plugins/catalog-graph/api-report.md

```diff
@@ -24,6 +24,7 @@ export const CatalogGraphPage: (props: {
     | {
         selectedRelations?: string[] | undefined;
         selectedKinds?: string[] | undefined;
+        selectedOwnedby?: string[] | undefined;
```

Comment:
The typical way we'd case this is `selectedOwnedBy`, note the capital `B`. Same thing with `ownedBy` below.

Reply:
Done